core/frontend/src/components/wizard/DefaultParamLoader.vue

```diff
@@ -172,6 +188,15 @@ export default Vue.extend({
         throw error
       }
     },
+    onLoadingTimeout() {
+      if (this.is_loading) {
+        this.loading_timeout_reached = true
+
+        if (this.selected_param_set_name === '') {
+          this.selected_param_set_name = this.not_load_default_params_option
+        }
```

Comment:
nice touch :)

Reply:
Can we do the same and skip the loading time when `invalid_board_or_version`? (maybe in another PR, since it's a different thing)